### ISSUE TITLE
Can you add support for SCP (secure copy over ssh) protocol to file manager

### ISSUE THREAD

**Issue by Mile-Lile** [linuxdeepin/dde-file-manager#419]
Can you add this usefull feature to file manager. Complementary to SSH, SCP is a secure way of transferring files between two remote hosts, a local host, and a remote host or vice versa. Usually used when transfering files from PC to e.g. router.
On other OSes it is used typing something like scp://192.168.1.1 and then pops up login like for ftp, smb, or ssh protocol... from shell it is used in this format> scp [options (optional)] source destination    -   username@hostname/IP_address:remote_directory or scp -P portnumber /destination/of/local/file.fileextension

**Comment by juteman** [comment]
You mean something like WinScp?

**Comment by Mile-Lile** [comment]
yes, but WinSCP is completelly standalone app... on Nautilus (GNOME file manager) on Ubuntu you just type scp://192.168.1.1 and you connect to remote host and do the copying of files... like ftp or ssh...

**Comment by Mile-Lile** [comment]
on deepin scp frpm cli is working no need to install anything, so I thaught maybe this wont be hard task for you... just to somehow enable it in file manager GUI... don't know :)

**Comment by Mile-Lile** [comment]
as a reference
[scp_on_debian](https://manpages.debian.org/buster/openssh-client/scp.1.en.htmll)

**Comment by Mile-Lile** [comment]
scp is using ssh for secure transfering file:

something like

`<html>
<body>


scp | [-346BCpqrTv] [-c cipher] [-F ssh_config] [-i identity_file] [-l limit] [-o ssh_option] [-P port] [-S program]
-- | --



</body>
</html>`

**Comment by justforlxz** [comment]
I think it will be more friendly to add the sshfs protocol, because it can open remote directories just like supporting smb and ftp protocols.

For example `sshfs://user:password@127.0.0.1/home`

Of course, you can also use sshfs manually, you can read the instructions.

@deepin-CD Do we support protocol extension plugins? I can develop this feature as a community.

**Comment by Mile-Lile** [comment]
well, sshfs is better... but... embed systems like Openwrt, ddwrt, EdgeOS etc. usually use dropbear not openssh.... dropbear doesn't support sshfs sadly... scp is wide spradly... but maybe both protocols could be added to file manager...
